src/routes/products/[id([0-9]+)]/edit.svelte

```diff
@@ -66,13 +66,23 @@
   let errorMessage = null;
   let warningDialogOpen = false;
 
-  async function addColor(color) {
-    let resp = await api.post('/colors', { data: { value: color } });
-    if (resp.status == 200) {
+  let colorDebounce = false;
+  async function addColor({ detail: color }) {
+    if (colorDebounce || colors.include(color)) {
```

Comment:
Your good sense of grammar caused a typo here
```suggestion
    if (colorDebounce || colors.includes(color)) {
```

Reply:
:)